tests/e2e/scripts/bootstrap_confirmations.py

```
@@ -0,0 +1,248 @@
+
+import sys
+from datetime import datetime, time, timedelta
+from pathlib import Path
+
+REPO_ROOT = Path(__file__).resolve().parents[3]
+SRC_DIR = REPO_ROOT / "src"
+if str(SRC_DIR) not in sys.path:
+    sys.path.insert(0, str(SRC_DIR))
+
+# pylint: disable=wrong-import-position  # Django imports must be after setup
+import django  # ruff:ignore[module-import-not-at-top-of-file]
```

Ruff (0.16.0)

[warning] Invalid rule code in suppression: module-import-not-at-top-of-file

Enable `lint.preview` to use rule names

(RUF102)